server/api/utils/clients/nuclio.py

```diff
@@ -237,7 +237,13 @@ def _send_request_to_api(
         if auth_info:
             auth = auth_info.to_nuclio_auth_info().to_requests_auth()
 
-        response = self._session.request(method, url, verify=False, auth=auth, **kwargs)
+        response = self._session.request(
+            method,
+            url,
+            verify=mlrun.config.config.httpdb.http.verify,
```

Comment:
General question - now all clients (`httpdb`, `nuclio`, `iguazio`) are using the same `http.verify` config parameter. Question is - do you see a situation where we want to use a different mode per client? For example, if I set this parameter to `True`, it assumes that all communication goes through SSL (i.e. through ingress) - is this always the case when calling `nuclio`/`iguazio`? 
Maybe it's better to set a different parameter per client? TBH, I'm not sure what the answer is...

Reply:
that is not a real case, it is either you have tls termination with valid cert or not